pkg/ansible/runner/runner_test.go

```diff
@@ -63,111 +51,88 @@ func checkCmdFunc(t *testing.T, cmdFunc cmdFuncType, role, playbook string, watc
 }
 
 func TestNew(t *testing.T) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Unable to get working director: %v", err)
+	}
+	validPlaybook := filepath.Join(cwd, "testdata", "playbook.yml")
+	validRole := filepath.Join(cwd, "testdata", "roles", "role")
 	testCases := []struct {
-		name  string
-		watch watches.Watch
```

Comment:
Why it was removed? Are the watches still working well after this change?

Reply:
I'm not certain I follow. The structure of the testCase struct was modified so that the watch could be constructed as it is [here](https://github.com/operator-framework/operator-sdk/pull/1852/files/532ebb705ce8856efc05ae64daba426c0a2d0ad8#diff-7160af983c29b9766f0f3db5e9bedd8cR130)